### PR TITLE
Add a stub for _Unwind_RaiseException

### PR DESCRIPTION
**Pull request by tomaka**
Just a basic implementation to make Rust happy.
This is the code I used to pass the tests. It's obviously not clean, but it's still better than aborting.

If I have some time I'll try figure out correctly how exceptions handling works in C++ and emscripten and write some correct code.

### CODE REVIEW

File: src/library.js [hunk 1]
```diff
@@ -4052,8 +4052,19 @@ LibraryManager.library = {
     return 0; // we cannot succeed
   },
 
+  _Unwind_RaiseException__deps: ['__cxa_find_matching_catch', '$EXCEPTIONS'],
   _Unwind_RaiseException: function(ex) {
-    abort('Unwind_RaiseException');
```
Comment (kripken) [inline src/library.js]: could this just call `cxa_throw`?

**Comment by kripken** [comment]
Also need a test for landing.

File: src/library.js (hunk 1 above)
Comment (tomaka) [inline src/library.js]: Probably yes, but `cxa_throw` takes three parameters and I'm not sure how to extract the relevant values from `ex`.

**Comment by tomaka** [comment]
Oh I was expecting to get away without a test :-/

File: src/library.js (hunk 1 above)
Comment (kripken) [inline src/library.js]: seems like you can call `cxa_throw(ex, null, null)`.

**Comment by kripken** [comment]
Sorry, yeah, we do need tests for stuff like this, or else we have no guarantee it won't break.

Let me know if you need any help writing one.

**Comment by kripken** [comment]
Might as well add a test later when things are further along. Merged.

**Pull request merged**